plutus-context-builder/plutus-context-builder.cabal

```diff
@@ -44,9 +44,10 @@ common lang
 
 library
   import:          lang
-  exposed-modules: Test.Plutus.ContextBuilder
+  exposed-modules:
+      Test.Plutus.ContextBuilder
+    , Test.Plutus.ContextBuilder.Internal
```

Comment:
I'd say 'minor version bump'. Also worth mentioning that everything in that module will need docs and `@since`s if they weren't docced or exposed before.

Reply:
I incremented the versions to 2.0.1 for plutus-context-builder and 9.1.1 to tasty-plutus. That's what minor means with PVP, right?